packages/graphql-plugin-auth/src/JWKSPlugin.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import jsonwebtoken from "jsonwebtoken";
+import JwksRsa, { JwksClient } from "jwks-rsa";
+import Debug from "debug";
+import { DEBUG_PREFIX } from "./constants";
+
+const debug = Debug(DEBUG_PREFIX);
+
+function verifyJWKS<T = any>({ client, token }: { client: JwksClient; token: string }): Promise<T> {
```

Comment:
**suggestion(non-blocking)**: Consider moving this function to the `JWKSPlugin` as a `private` method, this way all the related methods are packed within the class. And there would be no need to pass the client, as it is already in `this.` scope

Reply:
Sure